assets/app/actions/siteActions.js

```diff
@@ -65,7 +66,15 @@ export default {
 
   fetchSiteConfigsAndAssets(site) {
     return this.fetchSiteConfigs(site).then((site) => {
-      return this.fetchSiteAssets(site);
+      return this.fetchSiteAssets(site).then((site) => {
```

Comment:
I'm asking because this is a pretty slow part of the app, and we have to render 4 separate times before the user sees anything on the page.